build.sbt

```diff
@@ -74,9 +86,7 @@ lazy val macroVersion = "2.1.1"
 lazy val paradisePlugin = Def.setting{
```

Comment:
No longer necessary when only supporting 2.12+.

Reply:
I think it is still needed: https://travis-ci.org/julien-truffaut/Monocle/jobs/521607884